src/_pytest/terminal.py

```diff
@@ -998,7 +998,7 @@ def _get_line_with_reprcrash_message(config, rep, termwidth):
                     # u'😄' will result in a High Surrogate (U+D83D) character, which is
                     # rendered as u'�'; in this case we just strip that character out as it
                     # serves no purpose being rendered
-                    msg = msg.rstrip(u"\uD83D")
+                    msg = msg.rstrip("\uD83D")
```

Comment:
this does not do what you want, this will strip backslashes, u, D, 8, and 3 from the right side of the string